edx_proctoring/static/proctoring/js/views/proctored_exam_view.js

```diff
@@ -71,6 +76,22 @@ var edx = edx || {};
                 "credit eligibility in this course.\n");
```

Comment:
Possibly I commented on these strings in some PR but didn't actually get around to suggesting text because there were still open questions about the messaging around "violation" and "disqualification". 
I think the behavior is clearer now. If I remember correctly, navigating away from the exam in the same browser doesn't disqualify the student. The timer keeps running, and the exam is still in progress when the student returns to it. Is this correct? 
 @chrisndodge pls let me know if you want my suggested text in this PR or some other. Thanks.